frontend/src/features/permits/pages/Application/components/dashboard/StartApplicationAction.tsx

```diff
@@ -1,56 +1,161 @@
   const navigate = useNavigate();
-  const [chooseFrom, setChooseFrom] = useState<PermitType | typeof EMPTY_PERMIT_TYPE_SELECT>(
-    DEFAULT_PERMIT_TYPE
-  );
+  const [chooseFrom, setChooseFrom] = useState<
+    PermitType | typeof EMPTY_PERMIT_TYPE_SELECT
+  >(EMPTY_PERMIT_TYPE_SELECT);
 
-  const handleChooseFrom = (event: SelectChangeEvent) => {
-    setChooseFrom(event.target.value as PermitType | typeof EMPTY_PERMIT_TYPE_SELECT);
+  const [isError, setIsError] = useState<boolean>(false);
+
+  const handleChooseFrom = (
+    _event: React.MouseEvent<HTMLElement>,
+    item: PermitTypeChooseFromItem,
+  ) => {
+    setIsError(false);
+    setChooseFrom(item.value as PermitType);
+    handleClose();
   };
 
   const handleStartButtonClicked = () => {
     if (chooseFrom !== EMPTY_PERMIT_TYPE_SELECT) {
       navigate(APPLICATIONS_ROUTES.START_APPLICATION(chooseFrom));
+    } else {
+      setIsError(true);
     }
   };
 
+  // Update the structure of menuItems to ensure the callback is applied correctly
+  const menuItems = ALL_PERMIT_TYPE_CHOOSE_FROM_OPTIONS.map(
+    (item: PermitTypeChooseFromItem) => ({
+      ...item,
+      callback: (event: React.MouseEvent<HTMLElement>) =>
+        handleChooseFrom(event, item),
+      // Correctly set the nested item's callback
+      items: item?.items?.map((nestedItem) => ({
+        ...nestedItem,
+        callback: (event: React.MouseEvent<HTMLElement>) =>
+          handleChooseFrom(event, nestedItem),
+      })),
+    }),
+  );
+
+  const [anchorEl, setAnchorEl] = useState<HTMLDivElement | null>();
+  const open = Boolean(anchorEl);
+
+  const handleClick = (e: any) =>
+    setAnchorEl(e.currentTarget as HTMLDivElement);
+  const handleClose = () => setAnchorEl(null);
+
   return (
     <Box className="start-application-action">
-      <SelectPermitType
-        value={chooseFrom}
-        label={"Select Permit Type"}
-        onChange={handleChooseFrom}
-        menuItems={PERMIT_TYPE_CHOOSE_FROM_OPTIONS.map((data) => (
-          <MenuItem key={data.value} value={data.value}>
-            {data.label}
-          </MenuItem>
-        ))}
-      />
-
-      <Button
-        className="start-application-action__btn"
-        variant="contained"
-        onClick={handleStartButtonClicked}
-      >
-        Start Application
-      </Button>
+      <FormLabel className="start-application-action__label">
+        Select Permit Type
+      </FormLabel>
+      <div className="start-application-action__control">
+        <Tooltip
+          className="start-application-action__input-tooltip"
+          title={
+            chooseFrom !== EMPTY_PERMIT_TYPE_SELECT
+              ? getFormattedPermitTypeName(chooseFrom)
+              : EMPTY_PERMIT_TYPE_SELECT
+          }
+        >
+          <Button
+            className={`start-application-action__input ${open && "start-application-action__input--open"} ${isError && "start-application-action__input--error"}`}
```

Comment:
This className seems too long, you might want to extract it out into a variable and assign the variable here instead.

Reply:
Also please use the proper ternary syntax rather than the "&&" approach, especially here in string interpolations (ie. use ```open ? "..." : ""``` instead of ```open && "..."```)